task/clair-scan/0.1/clair-scan.yaml

```diff
@@ -43,22 +43,12 @@ spec:
     - name: test-format-result
       image: quay.io/redhat-appstudio/hacbs-test:latest
       script: |
+        . /utils.sh
         HACBS_ERROR_OUTPUT=$(jq -rc --arg date $(date +%s) --null-input \
           '{result: "ERROR", timestamp: $date}')
-        HACBS_TEST_OUTPUT=$(jq -rce --arg date $(date +%s) \
-          '.[] | { result: (if (.failures | length > 0) then "FAILURE" else "SUCCESS" end),
-                   timestamp: $date,
-                   namespace,
-                   successes,
-                   failures: (.failures | length)
-                 }' /tekton/home/clair-vulnerabilities.json || true)
+        HACBS_TEST_OUTPUT=
```

Comment:
Probably offtopic but should not we stop using HACBS keyword? So uste TEST_OUTPUT?

Reply:
That is a good point, but we will need to do that in concert with EC, so we're expecting to have collaborative story/ies to make the naming changes.

Reply:
Thank you. Let's update them to the name without `HACBS` once we make this decision in the future.